.eslintignore

```diff
@@ -0,0 +1,4 @@
+/dist
```

Comment:
shouldn't this be `dist/`

Reply:
Nope! We just want to exclude the `dist` directory in the root of the project.